### DISCUSSION TITLE
Extended Data Frames in LAWICEL / SLCAN

### DISCUSSION THREAD

**Discussion by Pacerino** [collin80/SavvyCAN#627] [General]
I have implemented SLCAN for my RejsaCAN board based on TWAI. During the development, however, I discovered that SavvyCAN does not process extended data frames with SLCAN. Is that the way it should be?

https://github.com/collin80/SavvyCAN/blob/0c20e1dd7989b64b1c4229ab4b4ab559ef37a5c7/connections/lawicel_serial.cpp#L477-L478

**Comment by make7535** [comment]
Perhaps this is the only problem for the implementation of lawicel in SavvyCAN and for me
